libs/src/oracle-sdk-v2/services/oraclev2/ethers/factory.ts

```diff
@@ -28,6 +28,7 @@ export type Config = {
   chainId: ChainId;
   url: string;
   address: string;
+  type: OracleType;
```

Comment:
so we can re-use the ethers client from OOv2, make oracle type part of the config.